igor-web/src/test/groovy/com/netflix/spinnaker/igor/jenkins/JenkinsBuildMonitorSpec.groovy

```diff
@@ -174,26 +174,27 @@ class JenkinsBuildMonitorSpec extends Specification {
 
     def 'should filter out builds older than look back window'() {
         given:
-        long stamp0 = System.currentTimeMillis() - (30 * 60 * 1000) // 30 minutes ago
-        long stamp1 = System.currentTimeMillis() - (10 * 60 * 1000) // 10 minutes ago
-        long stamp2 = System.currentTimeMillis() - (5 * 60 * 1000)  // 5  minutes ago
-        long stamp3 = System.currentTimeMillis()
+        long now = System.currentTimeMillis()
+        long nowMinus30min = now - (30 * 60 * 1000) // 30 minutes ago
+        long nowMinus10min = now - (10 * 60 * 1000) // 10 minutes ago
+        long nowMinus5min = now - (5 * 60 * 1000)  // 5  minutes ago
+        long durationOf5min = 300000
 
         and: 'a 6 minutes total lookBack window'
```

Comment:
this title isn't quite right anymore ... it's now `65` minutes right?

Reply:
Should be a 6 minute lookback window. The poll interval is 60 seconds, which is now correctly being translated for the lookback window calculation. In this test, the lookBackWindow is set to 5 minutes. Together, that gives 6 minutes.